utils.py

```diff
@@ -168,7 +168,39 @@ def __init__(self, name, parser, handler, args=[], kwargs={}, default_feed='full
         self.args = args
         self.kwargs = kwargs
 
+    def metadata(self, request):
+        meta = LazyBuilder(version='0.1.0')
+
+        meta.feeds.append(Feed(
+            name='today',
+            hour='8-14',
```

Comment:
nit: I know that this has been the default for a while, but it seem weird to fetch the full *and* today feed at 8 o'clock. I think `9-14` would make more sense.

Reply:
Afaik not all parsers include "today" in the "full" feed. I'd therefore like to leave the defaults for now.

It probably should be made configurable anyway.

Reply:
Ok, then let's leave this for now.